components/partials/Guzel.vue

```diff
@@ -0,0 +1,17 @@
+<script setup>
+</script>
+
+<template>
```

Comment:
[Normal]
Could you generate webinar cards on build from static file instead?
Then you would have only one component for webinar page.

Than you'd only need to edit one file to change the layout for all the webinars.